internal/operator/subscription.go

```diff
@@ -97,13 +98,18 @@ func (c operatorClient) ListPackageManifests(ctx context.Context, list *pkgserve
 	}
 
 	if len(filter) > 0 {
+		var notFound []string
 		for _, f := range filter {
 			for _, pkgm := range tmppkgmlist.Items {
 				if pkgm.Name == f {
 					list.Items = append(list.Items, pkgm)
+					notFound = append(notFound, f)
 				}
 			}
 		}
+		if len(notFound) > 0 {
+			return fmt.Errorf("Package %v not found in catalog source", notFound)
```

Comment:
@mrhillsman I'll add the catalog source name in the final draft, just want to make sure I'm on the right track or get advice on the proper idiom/format for all this